spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/ArmeriaAutoConfiguration.java

```diff
@@ -237,7 +238,12 @@ protected HttpResponse doGet(ServiceRequestContext ctx, HttpRequest req)
 
         final Server s = server.build();
 
-        s.start().join();
+        s.start().handle((result, throwable) -> {
```

Comment:
We commonly use `t` for the exception in a future handler.

Reply:
Thought we use `cause` more commonly? `t` is fine though.